tests/test_features/test_importances.py

```diff
@@ -93,22 +86,19 @@ def test_integration_coef(self):
         Integration test of visualizer with coef param
         """
 
-        concrete = self.load_data('concrete')
-        feats = ['cement','slag','ash','water','splast','coarse','fine','age']
-
-        # Create X and y datasets as numpy arrays
-        X = concrete[feats].copy()
-        X = X.view((float, len(X.dtype.names)))
-        y = concrete['strength']
+        # Load the test dataset
+        dataset = load_concrete(return_dataset=True)
+        X, y = dataset.to_numpy()
+        features = dataset.meta["features"]
```

Comment:
Whoo! New datasets module ftw!

Reply:
Agreed! I love how easy and readable the new oneliner style is, e.g. `X, y = load_dataset(return_dataset=True).to_numpy()`!